packages/slate-tools/tools/webpack/config/prod.js

```diff
@@ -104,7 +104,12 @@ module.exports = merge(
                 options: {
                   ident: 'postcss',
                   sourceMap: true,
-                  plugins: [autoprefixer, cssnano],
+                  plugins: [
+                    autoprefixer,
+                    cssnano({
```

Comment:
Want to do one better and make a config for the whole object in `slate-tools.config.js`?